modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro

```diff
@@ -294,6 +318,12 @@ definition {
 		Click.clickAtNotVisible(locator1 = "ObjectCustomViews#MARK_AS_DEFAULT");
 	}
 
+	macro resetFiltersOnEntries {
```

Comment:
Hey, @andreefarias Maybe this can be in the test level,  does this macro not is specific?